db/migrate/20160229131112_add_identifier_to_payments.rb

```diff
@@ -0,0 +1,5 @@
+class AddIdentifierToPayments < ActiveRecord::Migration
+  def change
```

Comment:
@nitin007 - Maybe I have missed it, but where are we using identifier in spree_payments.